Makefile

```diff
@@ -0,0 +1,89 @@
+###
+# This Makefile provides portable, stack-agnostic commands (unlike npm)
+#
+# These commands are meant to be run from your local machine, not from
+# within the docker container. Each command handles exec'ing into the
+# docker container to run the command
+
+ifneq (,)
+	This makefile requires GNU Make.
+endif
+# #######
+# Basics
+platform-onboarded:
+	make migrate-docker
+	docker exec -it platform_platform_1 /bin/bash
```

Comment:
what is the purpose of running just the shell here?

Reply:
I made changes here, Platform-onboarded, this command helps you get started, gets you on board with everything. so this command `docker-compose exec platform /bin/bash` takes you inside a platform docker container and the run `composer migrate` for setting up database connections, running migrations and seeders